hkdf/src/main/java/keywhiz/hkdf/Hkdf.java

```diff
@@ -66,15 +62,15 @@ private Hkdf(Hash hash, Provider provider) {
    * @return Hkdf constructed with default hash and derivation provider
    */
   public static Hkdf usingDefaults() {
-    return new Hkdf(DEFAULT_HASH, DEFAULT_PROVIDER);
+    return new Hkdf(DEFAULT_HASH, null);
```

Comment:
Provider field should be @Nullable now

Reply:
Update: we'll fix annotations when we find a replacement for findbugs and similar. We are thinking of using spotbugs as a replacement.